app/views/users/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="profile-box">
+    <div class="image-profile"><%= image_tag @user.photo, alt: 'User Photo' if @user.photo.present? %></div>
+    <div class="box">
+        <h2><%= @user.name %></h2>
+        <p><%= link_to "Post Counter:", user_posts_path(@user) %><%= @user.post_counter %></p>
+
+    </div>
+</div>
+
+<div class="bio">
+    <%= @user.bio %>
+</div>
+
+<ul>
+  <% @user.posts.each do |post| %>
```

Comment:
- [ ] Here, you should use the `recent_posts` method you created in your User model to display the 3 most recent posts.